README.md

```diff
@@ -0,0 +1,70 @@
+| xen_vman_default_vcpus                 | integer                 | `2`               | Number of virtual CPU cores to assign to a VM [¹](#xen_doc)                                                                           |     N    |
+| xen_vman_default_storage_type          | string                  | `nfs`             | Default filesystem backend to use                                                                                                     |     N    |
+| xen_vman_default_cpu_str               | string                  | `all,^1`          | List of which cpus the guest is allowed to use [¹](#xen_doc)                                                                          |     N    |
+| xen_vman_default_spice                 | `0` or `1`              | `0`               | Allow access to the display via the SPICE protocol [¹](#xen_doc)                                                                      |     N    |
+| xen_vman_default_spicehost             | string(IP-Address)      | `0.0.0.0`         | Specify the interface address to listen on if given, otherwise any interface [¹](#xen_doc)                                            |     N    |
+| xen_vman_default_spiceport             | integer(Port number)    | `3100`            | Specify the port to listen on by the SPICE server if the SPICE is enabled [¹](#xen_doc)                                               |     N    |
+| xen_vman_default_mouse                 | `0` or `1`              | `1`               | Whether SPICE agent is used for client mouse mode [¹](#xen_doc)                                                                       |     N    |
+| xen_vman_default_spicevdagent          | `0` or `1`              | `1`               | Enables spice vdagent, is automatically enabled when clipboard sharing is enabled [¹](#xen_doc)                                       |     N    |
+| xen_vman_default_share_clipboard       | `0` or `1`              | `1`               | Enables Spice clipboard sharing                                                                                                       |     N    |
+| xen_vman_default_max_usb_redirections  | integer(0-4)            | `4`               | Enables spice usbredirection                                                                                                          |     N    |
+| xen_vman_default_disks                 | list of strings         | `[]`              | Xen disc specification (see also [xl-disk-configuration.txt](http://xenbits.xen.org/docs/4.8-testing/misc/xl-disk-configuration.txt)) |     N    |
+| xen_vman_default_builder               | string                  | `generic`         | Select the domU guest type, valid values are `generic`(para virtualisation) and `hvm`(all hardware is emulated)                       |     N    |
```

Comment:
Why `generic` and not `pvm`?

Reply:
It's a xen thing, the default builder for a pvm is generic and hvm is for a hvm. You can write your own builders too, if you wish.

Reply:
That's weird, but it's great to use their standards 👍